assets/assets.go

```diff
@@ -14,11 +14,11 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package types
+package assets
 
-// AppMetadata stores serializable runtime REST access information
-type AppMetadata struct {
-	Node  string `yaml:"node"`
-	URL   string `yaml:"url"`
-	Debug bool   `yaml:"debug"`
-}
+import (
+	"embed"
+)
+
+//go:embed swagger/*
```

Comment:
Lets put the openapi yaml file alone using embed. For swagger files, we can possibly embed using the react (https://www.npmjs.com/package/swagger-ui-react) in our UI code.

Reply:
done